mockserver.js

```diff
@@ -134,31 +136,37 @@ var valueId = 16;
 var valueValueId = 106;
 var counterAttributeId = 3; // counter for ading attributes, for having unique IDs for each attribute
 
+// "Create" class diagram (does nothing except print debug info)
+app.put('/:username/classdiagram/:cdmName', (req, res) => {
+  console.log(`Create CDM request headers: ${JSON.stringify(req.headers)}`);
+  res.sendStatus(SUCCESS);
+});
+
 // GET class diagram
-app.get('/classdiagram/:cdmName', (req, res) => {
+app.get('/:username/classdiagram/:cdmName', (req, res) => {
```

Comment:
I updated the mock server while adding user support, for debugging purposes. In practice, the mock server is no longer usable, so it should be deprecated soon and eventually removed when WebCORE is easier to obtain, eg, as an executable JAR